pkg/core/mempool/mem_pool.go

```diff
@@ -403,6 +420,27 @@ func (mp *Pool) TryGetValue(hash util.Uint256) (*transaction.Transaction, bool)
 	return nil, false
 }
 
+// TryGetData returns data associated with the specified transaction if it exists in the memory pool.
+func (mp *Pool) TryGetData(hash util.Uint256) (interface{}, bool) {
+	mp.lock.RLock()
+	defer mp.lock.RUnlock()
+	if tx, ok := mp.verifiedMap[hash]; ok {
+		itm := item{txn: tx}
+		n := sort.Search(len(mp.verifiedTxes), func(n int) bool {
+			return itm.CompareTo(mp.verifiedTxes[n]) >= 0
```

Comment:
`== 0`?

Reply:
But this is a sort.Search, and `==0` is not a monotonous function. Am I missing something?

Reply:
Yeah, you're right, I'm looking at C# code too much these days probably.